Makefile

```diff
@@ -122,7 +122,10 @@ validate:
 	script/validate-c
 	$(GO) vet ./...
 
-ci: validate test release
+shellcheck:
+	shellcheck tests/integration/*.bats
```

Comment:
Should be inside `make validate`?

Reply:
Also, apt-get install shellcheck in Dockerfile?

Reply:
good point, fixed

Reply:
Note that there's upstream docker images, so instead of installing it in the dockerfile, you could also run a separate container just for shell check https://hub.docker.com/r/koalaman/shellcheck-alpine/tags

Reply:
We do not use shellcheck from inside a Docker container. We run `make ci` on the host (in Travis-CI environment that already guarantees that `shellcheck` is installed), and in the Makefile the `ci` target depends on `validate`, which does some checks on the host. It's the `test` target that uses dockerfile.

So, no need to install `shellcheck` from Dockerfile (or use a docker image with shellcheck).